tests/cypress/integration/reportRepair/communal.spec.js

```diff
@@ -60,7 +60,7 @@ describe('communal', () => {
           makeSelectionAndClickButton('No');
         });
         it('should redirect them to postcode then address page respectively',  () => {
```

Comment:
Should this be renamed since we are going to the tenant-or-leaseholder page?

Reply:
Renamed